webapp/src/main/java/org/apache/atlas/web/rest/EntityREST.java

```diff
@@ -1952,7 +1954,47 @@ public void repairAccessControlAlias(@PathParam("guid") String guid) throws Atla
        } finally {
               AtlasPerfTracer.log(perf);
        }
+    }
+
+
+    @POST
+    @Path("/{policyId}/link-business-policy")
+    @Produces(Servlets.JSON_MEDIA_TYPE)
+    @Consumes(Servlets.JSON_MEDIA_TYPE)
+    @Timed
+    public void linkBusinessPolicy(@PathParam("policyId") final String policyId, final LinkBusinessPolicyRequest request) throws AtlasBaseException {
+        if (!ARGO_SERVICE_USER_NAME.equals(RequestContext.getCurrentUser())) {
+            throw new AtlasBaseException(AtlasErrorCode.UNAUTHORIZED_ACCESS, RequestContext.getCurrentUser(), "Policy linking");
+        }
+        AtlasPerfTracer perf = null;
+        try {
+            if (AtlasPerfTracer.isPerfTraceEnabled(PERF_LOG)) {
+                perf = AtlasPerfTracer.getPerfTracer(PERF_LOG, "EntityREST.linkBusinessPolicy(" + policyId + ")");
+            }
+            entitiesStore.linkBusinessPolicy(policyId, request.getLinkGuids());
+        } finally {
+            AtlasPerfTracer.log(perf);
+        }
+    }
 
 
+    @POST
+    @Path("/{policyId}/unlink-business-policy")
+    @Produces(Servlets.JSON_MEDIA_TYPE)
+    @Consumes(Servlets.JSON_MEDIA_TYPE)
+    @Timed
+    public void unlinkBusinessPolicy(@PathParam("policyId") final String policyId, final LinkBusinessPolicyRequest request) throws AtlasBaseException {
+        if (!ARGO_SERVICE_USER_NAME.equals(RequestContext.getCurrentUser())) {
+            throw new AtlasBaseException(AtlasErrorCode.UNAUTHORIZED_ACCESS, RequestContext.getCurrentUser(), "Policy unlinking");
+        }
```

Comment:
Does RequestContext take from JWT token? Otherwise anyone can spoof that the username is Argo Service Username right? Or does it come after the authentication layer?